pulp_service/pulp_service/app/viewsets.py

```diff
@@ -0,0 +1,74 @@
+from pulpcore.app.viewsets import ContentGuardViewSet, RolesMixin
+
+from pulp_service.app.models import FeatureContentGuard
+from pulp_service.app.serializers import FeatureContentGuardSerializer
+
+
+class FeatureContentGuardViewSet(ContentGuardViewSet, RolesMixin):
+    """
+    Content guard to protect the content guarded by Subscription Features.
+    """
+
+    endpoint_name = "feature"
```

Comment:
Can you give me a URL example of interacting with this viewset? I think it's right, but I just was wanting an example.

Reply:
I'm using like this:
```
echo -n '{
"name": "test-feature-contentguard",
"description": ".",
"header_name": "x-rh-identity",
"jq_filter": ".agent_id",
"features": [
"RHEL-EUS-x86_64",
"RHEL-OS-x86_64"
]
}' | http POST :5001/pulp/another_domain/api/v3/contentguards/service/feature/